Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM nvidia/cuda:10.0-devel as builder
+FROM nvidia/cuda:11.0-devel as builder
```

Comment:
We could do this:

```
ARG CUDA=11.0
FROM nvidia/cuda:$CUDA-devel as builder

...

FROM nvidia/cuda:$CUDA-base
```

And then make images with a different cuda version using `--build-args`, we do this on `hathor-core` to generate images with different python versions.

That would make it possible for old GPUs to still be used. Although I'm not sure if it's actually useful to support these GPUs, it seems simple enough. What do you think?

Reply:
Yeah, I think this is a great idea.

I'll do this when I implement the image generation job then. We can leave this PR on hold in the meantime.

Reply:
Great!